### ISSUE TITLE
Incorporate DTI-TK into our evaluation of registration methods

### ISSUE THREAD

**Issue by ebrahimebrahim** [brain-microstructure-exploration-tools/abcd-registration-experiments#4]
https://dti-tk.sourceforge.net/pmwiki/pmwiki.php

**Comment by ebrahimebrahim** [comment]
Still making sure we can save the warps. It will also be important to have the warps in a format that mrtrix can accept.

**Comment by ebrahimebrahim** [comment]
Update: It seems that figuring out exactly how DTI-TK is storing and using warps is a massive pain. Some options:
- Keep banging head on wall till we figure it out. Submit request for source code possibly.
- Talk to Gary?
- Talk to Martin about whether they have used the warps in their group. They use DTI-TK for sure, but IDK if they have used the warps directly.
- Give up because if it's that hard to understand the warp format then this is not a good option anyway and doesn't need to be part of our evaluation and recommendations.

**Comment by ebrahimebrahim** [comment]
(didn't mean to close this yet)

**Comment by ebrahimebrahim** [comment]
@jamesfishbaugh When you get a chance please include your notes on this tricky dti-tk thing

**Comment by ebrahimebrahim** [comment]
Made a few tweaks in #34, but something seems to be not working here. It runs successfully, but when eyeballing the results to check for alignment it does not seem that things are getting pushed into alignment. To test this, I used the `diffeo_forward.nii.gz` that gets written out by `pairwise_evaluation_dtitk.py` to deform the source FA image. I compared the deformed source FA to the target FA. The deformed FA was a bit oddly shaped and did not appear to be more aligned to the target FA than the original source FA.

Here is how I used `diffeo_forward.nii.gz` when my `pairwise_evaluation_dtitk.py` output folder was `test_dtitk_script`:
```sh
WARPFILE="test_dtitk_script/registration/diffeo_forward.nii.gz"
DEFORMATION="./mrtrix_forward_diffeo.nii.gz"
warpconvert $WARPFILE displacement2deformation $DEFORMATION -force
FA="input-data/6NU2WWNR/fa/sub-NDARINV6NU2WWNR_ses-2YearFollowUpYArm1_run-01_dwi_fa.nii.gz"
FA_WARPED="./warped_fa.nii.gz"
mrtransform $FA $FA_WARPED -warp $DEFORMATION -interp cubic
```

I am thinking let's skip dti-tk. It's old anyway. At this point our work on it is driven largely by the sunk cost fallacy.

And we already will have a comparison with a state of the art non deep learning method that is properly using the rich DWI information: the mrtrix fod based method by Raffelt! So let's just make sure to include that in evaluation.
